core/src/test/java/org/elasticsearch/search/aggregations/metrics/geobounds/InternalGeoBoundsTests.java

```diff
@@ -102,4 +103,54 @@ protected void assertFromXContent(InternalGeoBounds aggregation, ParsedAggregati
     protected Writeable.Reader<InternalGeoBounds> instanceReader() {
         return InternalGeoBounds::new;
     }
+
+    @Override
+    protected InternalGeoBounds mutateInstance(InternalGeoBounds instance) {
+        String name = instance.getName();
+        double top = instance.top;
+        double bottom = instance.bottom;
+        double posLeft = instance.posLeft;
+        double posRight = instance.posRight;
+        double negLeft = instance.negLeft;
+        double negRight = instance.negRight;
+        boolean wrapLongitude = instance.wrapLongitude;
+        List<PipelineAggregator> pipelineAggregators = instance.pipelineAggregators();
+        Map<String, Object> metaData = instance.getMetaData();
+        switch (between(0, 8)) {
+        case 0:
+            name += randomAlphaOfLength(5);
+            break;
+        case 1:
+            top += between(1, 20);
+            break;
+        case 2:
+            bottom += between(1, 20);
+            break;
+        case 3:
+            posLeft += between(1, 20);
+            break;
+        case 4:
+            posRight += between(1, 20);
+            break;
+        case 5:
+            negLeft += between(1, 20);
+            break;
+        case 6:
+            negRight += between(1, 20);
+            break;
+        case 7:
+            wrapLongitude = wrapLongitude == false;
+            break;
+        case 8:
+        default:
```

Comment:
out of curiosity: why default is also the last case? shouldn't we rather be strict and throw exception if the randomization goes out of the supported bounds?